src/main/resources/static/_shared.yml

```diff
@@ -4903,6 +4908,47 @@ components:
         - outOfServiceTo
         - reason
         - daysLost
+    Cas1OutOfServiceBedHistoryItem:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        updatedAt:
+          type: string
+          format: date-time
+        updatedBy:
+          $ref: '#/components/schemas/User'
+        historyType:
+          type: array
+          items:
+            $ref: '#/components/schemas/Cas1OutOfServiceBedHistoryType'
+        outOfServiceFrom:
+          type: string
+          format: date
+        outOfServiceTo:
+          type: string
+          format: date
+        reason:
+          $ref: '#/components/schemas/Cas1OutOfServiceBedReason'
+        referenceNumber:
+          type: string
+        notes:
+          type: string
+      required:
+        - id
+        - updatedAt
+        - updatedBy
+        - historyType
+    Cas1OutOfServiceBedHistoryType:
```

Comment:
would Cas1OutOfServiceBed{Change,Update}Type be clearer? i.e. map to the `Cas1OutOfServiceBed{Change,Update}Entity` name?